keras/src/layers/preprocessing/image_preprocessing/bounding_boxes/converters_new.py

```diff
@@ -0,0 +1,102 @@
+from keras.src.api_export import keras_export
```

Comment:
Why converters_new? why not just converters?

Reply:
Yes, just wanted a initial review so that I can replace the existing one.